ospfd/ospf_asbr.c

```diff
@@ -134,9 +134,9 @@ ospf_external_info_add(struct ospf *ospf, uint8_t type, unsigned short instance,
 				  INET6_BUFSIZ);
 			if (IS_DEBUG_OSPF(lsa, LSA_GENERATE))
 				zlog_debug(
-					"Redistribute[%s][%d][%u]: %s/%d discarding old info with NH %s.",
+					"Redistribute[%s][%d][%u]: %pI4/%d discarding old info with NH %s.",
 					ospf_redist_string(type), instance,
-					ospf->vrf_id, inet_ntoa(p.prefix),
+					ospf->vrf_id, &p.prefix,
```

Comment:
maybe we should replace `%s/%d` pairs to `%pFX`?

Reply:
yes, quite right. I've caught a couple just now, I'll keep looking for more...